src/test/bin/winkernel/control.cpp

```diff
@@ -82,6 +84,8 @@ QuicTestCtlInitialize(
     WDF_IO_QUEUE_CONFIG QueueConfig;
     WDFQUEUE Queue;
 
+    QUIC_ENABLE_PRIVATE_NMR_PROVIDER();
```

Comment:
You have to have two versions. Normal tests still need to use the normal MMR interface, because that's what's tested inbox.